CBI.Jenkinsfile

```diff
@@ -102,14 +112,18 @@ spec:
       steps {
           sh """
             /home/vnc/.vnc/xstartup.sh
-            ./1-maven-build.sh -s /home/jenkins/.m2/settings.xml --tp=${params.target_platform} --local-repository=/home/jenkins/.m2/repository
+            ./1-maven-build.sh -s /home/jenkins/.m2/settings.xml \
+              --tp=${params.target_platform} \
+              --local-repository=/home/jenkins/.m2/repository \
```

Comment:
is the trailing `/ `intended

Reply:
no. does not harm, but should be removed. I'll update this.